README.md

```diff
@@ -91,12 +92,33 @@ prefixedEnv := &settings.PrefixSource{
     Prefix: []string{"APP"},
 }
 // Create an ordered lookup where ENV is first with a fallback to
-// YAML which can fallback to JSON.
+// YAML which can fallback to JSON.  The first found is used.
 finalSource := []settings.MultiSource{prefixedEnv, yamlSource, jsonSource}
 
 v, found := finalSource.Get(context.Background(), "setting")
 ```
 
+`MultiSource` supports variable substitution, so that you can effectively perform
```

Comment:
Yup.  Agree that this would/could be a good feature.  I'm inspired to make a followup PR for that, being careful not to break existing expected behavior.  TDD, characterization tests FTW.

Reply:
Also I realize I need to change the behavior in this PR to do the latter in my original PR description for exactly "preserve current observed behavior" backward compat.